PureLayout/PureLayout/ALView+PureLayout.h

```diff
@@ -118,6 +120,20 @@ __PL_ASSUME_NONNULL_BEGIN
 /** Pins an edge of the view to a given edge of another view with an offset as a maximum or minimum. */
 - (NSLayoutConstraint *)autoPinEdge:(ALEdge)edge toEdge:(ALEdge)toEdge ofView:(ALView *)otherView withOffset:(CGFloat)offset relation:(NSLayoutRelation)relation;
 
+/** Pin a space between self and the other view [self]-space-[otherView], either on the horizontal or vertical axis. */
+- (NSLayoutConstraint *)autoPinSpace:(CGFloat)space followedByView:(ALView *)otherView onAxis:(ALAxis)axis;
+
+/** Pin a space between self and the other view [self]-space-[otherView], either on the horizontal or vertical axis with the space as a maximum or minimum.  */
+- (NSLayoutConstraint *)autoPinSpace:(CGFloat)space followedByView:(ALView *)otherView onAxis:(ALAxis)axis relation:(NSLayoutRelation)relation;
+
+/** Pin trailing to leading with a space H:[self]-space-[otherView] */
+- (NSLayoutConstraint *)autoPinHorizontalSpace:(CGFloat)space followedByView:(ALView *)otherView;
+
+/** Pin bottom to top with a space V:[self]-space-[otherView] */
+- (NSLayoutConstraint *)autoPinVerticalSpace:(CGFloat)space followedByView:(ALView *)otherView;
```

Comment:
These 4 pin space methods just seem to be a different way of naming the existing `autoPinEdge:toEdge:ofView:withOffset:` methods. I have a few shortcut methods like this myself, I just have another category on UIView in my projects that I put them in and they call to the existing PureLayout methods. I don't think these sort of things make sense to add for everyone. And in this case you can just use VFL for certain constraints if you prefer it.